### PR TITLE
feat: dialog docs

### PR DESCRIPTION
**Pull request by markuczy**

### CODE REVIEW

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+import {
+  providePortalDialogService
+} from '@onecx/portal-integration-angular';
+
+...
+providers: [
+    providePortalDialogService(),
+]
+...
+----
```
Comment (BenjaminPabst) [inline docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:23]: maybe note this has to be done in the module? Or in which file it needs to be respectively (e.g., standalone?)

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+== Importing PortalDialogService
+To properly use PortalDialogService, please import it via the `providePortalDialogService`.
+
+.Provide PortalDialogService via the providers array
+[source,javascript]
+----
+import {
+  providePortalDialogService
+} from '@onecx/portal-integration-angular';
```
Comment (bastianjakobi) [inline docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:16]: At this point do we still want to publish guides referring to PIA or would now be a good time to move the dialog service and deprecate it in PIA?

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+|`componentOrMessage` |`Type<any> \| Type<DialogResult<T>> \| Component<T> \| TranslationKey \| DialogMessage` | Either a component or a translation key of a message with optional parameters and icon to be displayed next to the message.
+|`primaryButtonTranslationKeyOrDetails` |`TranslationKey \| ButtonDialogButtonDetails` | Translation key with optional parameters and icon to be displayed next to the text of the button.
+|`secondaryButtonTranslationKeyOrDetails?` |`TranslationKey \| ButtonDialogButtonDetails` | Translation key with optional parameters and icon to be displayed next to the text of the button.
+|`extras?` |`PortalDialogConfig` | Configuration object allowing for customization of the dialog behavior and visual aspects.
+|===
+
+.Display dialog with message
+[source,javascript]
+----
+this.portalDialogService.openDialog('TITLE_KEY', 'MESSAGE_KEY', 'PRIMARY_BUTTON_KEY', 'SECONDARY_BUTTON_KEY').subscribe((stateOnClose) => {
```
Comment (bastianjakobi) [inline docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:58]: Should we mention subscription cleanup here, or is this irrelevant/somehow handled automatically?

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc (hunk 2 above)
Comment (markuczy) [reply docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:16]: I agree, however maybe we could wait for libs v6 release candidate for this?

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc (hunk 3 above)
Comment (markuczy) [reply docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:58]: I think its crucial to include the part that you actually need to subscribe, but cleanup is not related to the dialog itself working correctly I think. Dev's should know to cleanup subs from my pov

File: docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc (hunk 2 above)
Comment (KimFFVII) [reply docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc:16]: There is a story for moving all the useful stuff from PIA such as the dialog service to their correct places. So I would say we can wait until then.

**Pull request merged**